### PR TITLE
Raw10/12 CTS tests

### PR DESCRIPTION
**Pull request by JohnKesapidesARM**
Add support for cl_ext_image_raw10_raw12 testing
on test_image_streams.

### CODE REVIEW

File: test_common/harness/imageHelpers.h [hunk 1]
```diff
@@ -255,10 +291,22 @@ void read_image_pixel(void *imageData, image_descriptor *imageInfo, int x,
...
     // Advance to the right spot
     char *ptr = (char *)imageData;
-    size_t pixelSize = get_pixel_size(format);
-
-    ptr += z * slice_pitch_lod + y * row_pitch_lod + x * pixelSize;
-
+    switch (format->image_channel_data_type)
+    {
+        case CL_UNSIGNED_INT_RAW10_EXT: {
+            ptr += z * slice_pitch_lod + y * row_pitch_lod + (x / 4) * 5;
```
Comment (Jeremy-Kemp) [inline test_common/harness/imageHelpers.h:297]: Can we use `RAW10_EXT_CLUMP_SIZE` and `RAW10_EXT_CLUMP_NUM_PIXELS` throughout instead? (Same for raw12)

File: test_conformance/images/kernel_read_write/main.cpp [hunk 2]
```diff
@@ -385,6 +393,7 @@ test_definition test_list[] = {
     ADD_TEST_VERSION(image_from_small_buffer_negative, Version(3, 0)),
     ADD_TEST_VERSION(image_from_buffer_fill_positive, Version(3, 0)),
     ADD_TEST_VERSION(image_from_buffer_read_positive, Version(3, 0)),
+    ADD_TEST_VERSION(cl_ext_image_raw10_raw12, Version(3, 0)),
```
Comment (Jeremy-Kemp) [inline test_conformance/images/kernel_read_write/main.cpp:396]: This extension could be supported from a 1.2 implementation.

File: test_common/harness/imageHelpers.h (hunk 1 above)
Comment (JohnKesapidesARM) [reply test_common/harness/imageHelpers.h:297]: sure, I'll make the change

**Review by Jeremy-Kemp — APPROVED** [review]
Thanks for making the changes, apologies for the delay!

**Review by lakshmih — APPROVED** [review]

**Comment by bashbaug** [comment]
Merging as discussed in the May 21st teleconference.

**Pull request merged**